src/v/cluster/tests/leader_balancer_test_utils.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2022 Vectorized, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+
+#include "cluster/scheduling/leader_balancer_greedy.h"
+
+namespace leader_balancer_test_utils {
```

Comment:
I wasn't really sure about the namespace here. Our tests don't seem to be in explicit namespaces at all, so not too sure about util methods like this? I just used a unique namespace named after the test.